UK/Area/London TC/TC East.prf

```diff
@@ -22,7 +22,7 @@ RecentFiles	Recent3	\..\..\Data\ASR\LTC\TCE3.asr
 RecentFiles	Recent4	\..\..\Data\ASR\LTC\TCE4.asr
 RecentFiles	Recent5	\..\..\Data\ASR\LTC\Stansted SMR.asr
 RecentFiles	Recent6	\..\..\Data\ASR\LTC\TCE_Ground.asr
-Settings	sector	\..\..\Data\Sector\UK_2023_08.sct
```

Comment:
@chssn The directory itself should not be changed, only the file name. Keeping the directories relative, rather than fixed keeps the pack portable, for those users who like to put the files in a different location.

Reply:
You are indeed correct!